src/ontology/mondo-ingest.Makefile

```diff
@@ -72,6 +72,7 @@ $(COMPONENTSDIR)/omim.owl: $(TMPDIR)/omim_relevant_signature.txt | component-dow
 			--update ../sparql/fix-labels-with-brackets.ru \
 			--update ../sparql/fix_illegal_punning_omim.ru \
 			--update ../sparql/exact_syn_from_label.ru \
+			--update ../sparql/convert-OMO_0003000-to-MONDO_ABBREVIATION.ru \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/omim.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/omim.owl -o $@; fi
```

Comment:
[Trish observed](https://github.com/monarch-initiative/mondo-ingest/pull/666#discussion_r1769647426) unintended consequences of a harmful `robot remove` line that I'd tested out and added here and forgot to remove. I've since removed and am rerunning builds.

Reply:
RE Trish:
> Changes and build look good

Sounds good. Gonna hold off on merging until #663 is approved, as this PR of necessity includes those changes.